src/components/addDiceModal.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import './addDiceModal.css'
+
+const faces = [
+  4,
+  6,
+  10,
+  20,
+]
+
+const AddDiceModal = ({ addDice }) => {
+  return (
+    <div className="modal">
+      <div className="card">
```

Comment:
A way to leave the modal without adding a dice is missing

Reply:
I'll use https://github.com/streamich/react-use/blob/master/docs/useOutsideClick.md
+ a close button